{"name": "# C5163 Device Locations section", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5163_Device_Locations_section.py\", line 8, in dvc_lctns_txt_hr\n    context.app.main_page.dvc_lctns_txt_hr(dvc_lctns_txt_hr)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 975, in dvc_lctns_txt_hr\n    dvc_lctns_txt_hr = wait.until(EC.visibility_of_element_located(DVC_LCTNS_TXT_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1605386412538, "stop": 1605386416822}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1605386416822, "stop": 1605386419641}, {"name": "Then Click Dashboard menu icon and make sure Dashboard page reloaded", "status": "passed", "start": 1605386419641, "stop": 1605386419766}, {"name": "And Make sure Device Locations words are on the screen", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5163_Device_Locations_section.py\", line 8, in dvc_lctns_txt_hr\n    context.app.main_page.dvc_lctns_txt_hr(dvc_lctns_txt_hr)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 975, in dvc_lctns_txt_hr\n    dvc_lctns_txt_hr = wait.until(EC.visibility_of_element_located(DVC_LCTNS_TXT_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1605386419766, "stop": 1605386437966}, {"name": "And Mouse hover question mark in top right of the block and make sure tooltip appears", "status": "skipped", "start": 1605386440073, "stop": 1605386440073}, {"name": "Then Make sure that click on Device Locations drop-down list is working", "status": "skipped", "start": 1605386440073, "stop": 1605386440073}], "start": 1605386407220, "stop": 1605386440073, "uuid": "8c6b1050-dcf2-4171-954a-bd5c2aa3fe05", "historyId": "b89a768134587347e4e0ce9a72fe873a", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5163 Device Locations section"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}